models/CalculationModel.java

```diff
@@ -0,0 +1,51 @@
+package models;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class CalculationModel {
+	private String selectedDay;
+	private String workBegin;
+	private String workEnd;
+	private String totalWorkTime;
+	private String totalBreakTime;
+	
```

Comment:
The parts that are there are fine to me. I'm just missing overtime/flexiTime

Reply:
@valientes  will be added when we get the plannedWorkingTime from Database :)

Reply:
@jerdbaer ah, I remember now we talked about that earlier today. My memory is a mess rn...